### PR TITLE
Allows to see ILM failures for hidden and system indices

### PR DESCRIPTION
**Pull request by lucabelluccini**
The changes should allow to see ILM failures for system/hidden indices.

### CODE REVIEW

File: src/main/resources/elastic-rest.yml [hunk 1]
```diff
@@ -357,11 +357,13 @@ ilm_explain:
   subdir: "commercial"
   versions:
     ">= 6.6.0": "/*/_ilm/explain?human&pretty"
+    ">= 7.7.0": "/*/_ilm/explain?human&pretty&expand_wildcards=all"
 
 ilm_explain_only_errors:
   subdir: "commercial"
   versions:
     ">= 7.4.0": "/*/_ilm/explain?only_errors=true&human&pretty"
+    ">= 7.7.0": "/*/_ilm/explain?only_errors=true&human&pretty&expand_wildcards=all"
```
Comment (jpcarey) [inline src/main/resources/elastic-rest.yml:366]:
```suggestion
    ">= 7.4.0 < 7.7.0": "/*/_ilm/explain?only_errors=true&human&pretty"
    ">= 7.7.0": "/*/_ilm/explain?only_errors=true&human&pretty&expand_wildcards=all"
```

File: src/main/resources/elastic-rest.yml [hunk 2]
```diff
@@ -357,11 +357,13 @@ ilm_explain:
   subdir: "commercial"
   versions:
     ">= 6.6.0": "/*/_ilm/explain?human&pretty"
+    ">= 7.7.0": "/*/_ilm/explain?human&pretty&expand_wildcards=all"
```
Comment (jpcarey) [inline src/main/resources/elastic-rest.yml:360]:
```suggestion
    ">= 6.6.0 < 7.7.0": "/*/_ilm/explain?human&pretty"
    ">= 7.7.0": "/*/_ilm/explain?human&pretty&expand_wildcards=all"
```

**Review by jpcarey — CHANGES_REQUESTED** [review]

**Comment by lucabelluccini** [comment]
Accepted the modifications (thank you for the fixes and review @jpcarey)!

**Review by jpcarey — APPROVED** [review]
LGTM. @lucabelluccini I didn't realize this was ready to re-review. It does not show up in my Github PR view unless it is awaiting review / re-review.

**Pull request merged**